internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,42 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"project/internal/spentcalories"
+	"strconv"
+	"strings"
 	"time"
 )
 
 var (
-	StepLength = 0.65 // длина шага в метрах
+	StepLength     = 0.65 // длина шага в метрах
+	SliceLenghtErr = errors.New("Неверная длина слайса")
+	StrToIntErr    = errors.New("Ошибка преобразования string в int")
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	slice := strings.Split(data, ",")
+	if len(slice) != 2 {
+		return 0, 0, SliceLenghtErr
+	}
+
+	steps, err := strconv.Atoi(slice[0])
+	if err != nil {
+		return 0, 0, StrToIntErr
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```